client/src/pages/_app.tsx

```diff
@@ -58,7 +58,7 @@ const client = new ApolloClient({
     typePolicies: {
       Query: {
         fields: {
-          paginatedEvents: offsetLimitPagination(),
+          paginatedEventsWithTotal: offsetLimitPagination(),
```

Comment:
The issue do happen when navigating from home to events too, but doesn't happen when navigating from chapters to home and vise versa.

I think apollo didn't understand that the data we are fetching is the same as data that it cached, because I didn't define the params. 

Kinda like, hey these objects aren't the same as `paginatedEvents(offset, limit)`, so let's refetch the same data with the same keys, instead of using the cached data.

Which result in having two of the same object with the same key as reference, and React warning us about this.

This is pure guess from reading the docs, which I misread a lot. I haven't seen an actual code.